docs/technologies.md

```diff
@@ -71,6 +71,13 @@ separate things. Your Beam process might need just a subset of the project requi
 options.view_as(SetupOptions).requirements_file = resolve(get_resource_absolute_path('my-beam-process-requirements.txt', Path(__file__)))
 ```
 
+Note that in those lines:
```

Comment:
Say what you want to say first, and then, provide a code sample (not in the middle of the sentence).